src/scriptworker/constants.py

```diff
@@ -67,6 +67,7 @@
         "task_log_dir": "...",  # set this to ARTIFACT_DIR/public/logs
         "artifact_upload_timeout": 60 * 20,
         "max_concurrent_downloads": 5,
+        "provisionprofile_dir": "...",  # For apple signing
```

Comment:
This looks like an iscript script_config.yml setting, not a scriptworker worker daemon config setting, no?